playwright.config.ts

```diff
@@ -20,7 +20,7 @@ export default defineConfig({
         ['html', { outputFolder: 'playwright-report' }],
       ]
     : 'list',
-  timeout: process.env.CI ? 60000 : 30000,
+  timeout: process.env.CI ? 300000 : 30000,
```

Comment:
Why should we change this globally? This will make failing too slow - e.g. if you have failing test, you will be waiting for results significantly longer.

Reply:
overriding that locally like this `await status.waitFor({ state: 'visible', timeout: 300000 });` didn't seem to work, let me try test.setTimeout

Reply:
You can adjust the timeout for your specific test by passing in the value to the expect method.
expect (something).doesAThing({timeout:1234})